yarn.lock

```diff
@@ -2459,10 +2459,8 @@
   dependencies:
     defer-to-connect "^2.0.1"
 
-"@terraware/web-components@^1.0.30":
-  version "1.0.30"
-  resolved "https://registry.yarnpkg.com/@terraware/web-components/-/web-components-1.0.30.tgz#837d1c14ddeecec8c4592e2b2dceb362c3aebcda"
-  integrity sha512-6LiUClJTuJYBC+X6/hkOkHIhsbqbSPTl8gur2AWxLp4JMPNzTu8eSR0IujEw8GLoOpWAFBKFgXJTHprODC2CnA==
+"@terraware/web-components@file:../web-components/dist":
```

Comment:
This should be 31, you can pick up changes running `yarn`. If you copied over the dist into node_modules for testing (which is what I do a lot).. run `yarn add @terraware/web-components` .. that will update this file to show 31.

Reply:
Yeah missed that thanks.  FYI in case you didn't know, in your package.json you can use `"file: <path to dist>"` instead of copying over.  Still have to run `yarn install --checkfiles` to get the latest, but easier than manually copying.